### DISCUSSION TITLE
How does YALMIP pre-allocate memory to constraints?

### DISCUSSION THREAD

**Discussion by mingluo-feng** [yalmip/YALMIP#1432] [Problems using YALMIP]
Hello, I have defined a very large number of constraints using YALMIP and the process has taken a long time without pre-allocating memory. Is there any way to pre-allocate memory when constructing constraints for optimisation models using YALMIP? Thank you!

**Comment by johanlofberg** [comment]
There is no such thing. If you have poor performance, you've either vectorized poorly, or you are creating complicated expressions. Impossible to know without reproducible code

**Reply by mingluo-feng** [reply]
OH, thank you very much. Now there's a direction of improvement for me.
